Update the workflow process

dbrower opened the pull request:
also update formatting so more items are hot-linked.

ericleasemorgan commented:
Don, "Tastes great. Less filling." 

There is one small point. People can join the Slack channel sans an invitation; people can join the Slack channel with following URL:

  http://bit.ly/distantreader-on-slack

Please edit accordingly, and I'll commit sooner rather than later. 

P.S. Please remember, such documents like the CONTRIB are "living" documents. Be forewarned?  :)

--
Eric

dbrower commented:
No problem, I've added the slack link. I also split out a GUIDE from the contributing, since they have two different purposes.

Of course these documents are living...?
